src/App/routes.jsx

```diff
@@ -87,6 +87,12 @@ export default [
     ),
     path: '/purge-caches',
   },
+  {
+    component: loadable(() =>
+      import(/* webpackChunkName: 'CachePurges' */ '../views/Quickstart')
```

Comment:
s/CachePurges/QuickStart/

Reply:
Ugh, I keep doing the same silly mistake.